src/app/app-extras.module.ts

```diff
@@ -37,6 +41,8 @@ import {
     SkyModalModule,
     SkySummaryActionBarModule,
     SkyTabsModule,
+    SkySplitViewModule,
+    SkyConfirmModule,
```

Comment:
Does your split view demo necessarily need the `SkyConfirmModule`?

Reply:
Yes. It has a feature in it that pops a confirm modal.